Firmware/Marlin_main.cpp

```diff
@@ -8436,7 +8397,7 @@ SERIAL_PROTOCOLPGM("\n\n");
     case 702:
     {
         float z_target = 0;
-        float unloadLength = FILAMENTCHANGE_FINALRETRACT;
```

Comment:
Why hard code it here as you changed these value in the variants?

Reply:
That's a good point. I reverted it in [e24c220](https://github.com/prusa3d/Prusa-Firmware/pull/4193/commits/e24c2200f12e9ead1143372a8067c77ac52d45ee).